deployments/fargate/README.md

```diff
@@ -0,0 +1,143 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html)
+for further reading.
+
+Unless stated otherwise, the
+[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed to an ECS task in a **sidecar** container. We are using the term
+"sidecar" loosely here to mean having the Collector container as a sidecar to multiple or
+all monitored application containers in the task. We do not mean a Collector sidecar container
+per monitored application container as in the strict sense.
+
+Requires Connector release v0.30.0 or newer which corresponds to image tag 0.30.0 and newer.
+See image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+## Getting Started
+The default configuration file is located at `/etc/otel/collector/fargate_config.yaml`
+in the Collector image according to the Collector image Dockerfile
+[here](../../cmd/otelcol/Dockerfile). See the contents of the default configuration file
+[here](../../cmd/otelcol/config/collector/fargate_config.yaml). Note that receivers
+`hostmetrics` and `smartagent/ecs-metadata` are specified by default.
```

Comment:
I think we should have a task definition example as provided in signalfx-agent https://github.com/signalfx/signalfx-agent/blob/main/deployments/fargate/example-fargate-task.json . Not necessary within this PR, probably can be another one.

Reply:
Added container definition JSON for the collector